### PR TITLE
Add support for microsoft edge

### PR DESCRIPTION
**Pull request by paladox**
I've had this fork of this extension here https://github.com/paladox/EdgeWikimediaDebug/ just moving it to this repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,5 @@ inject the header into Wikimedia HTTP/S requests. Header injection can be
 toggled via toolbar icon.
 
 You can install it for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/wikimedia-debug-header/), or
-[Chromium](https://chrome.google.com/webstore/detail/wikimediadebug/binmakecefompkjggiklgjenddjoifbb).
+[Chromium](https://chrome.google.com/webstore/detail/wikimediadebug/binmakecefompkjggiklgjenddjoifbb) or
+[Edge]().
```
Comment (bd808) [inline README.md:10]: You have added a link with no target URL

File: manifest.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
         }
     ],
     "background": {
-        "scripts": ["background.js"]
```
Comment (bd808) [inline manifest.json:36]: Is the scripts shortcut not supported by Edge?

**Review by bd808 — CHANGES_REQUESTED** [review]

File: manifest.json (hunk 2 above)
Comment (paladox) [reply manifest.json:36]: Not sure, there converted replaced this I think.

File: README.md (hunk 1 above)
Comment (paladox) [reply README.md:10]: Yep as it is not currently published.

Comment (bd808) [reply README.md:10]: So it probably shouldn't be a link?

File: manifest.json (hunk 2 above)
Comment (bd808) [reply manifest.json:36]: Can you do some research to see if "scripts" would work just as well?

**Comment by Krinkle** [comment]
Closing as the next Edge release will be Chromium-based and thus support the same extensions directly.

**Pull request closed without merging**